packages/core/types/functions.d.ts

```diff
@@ -130,7 +134,7 @@ export type CreatePerformGet<
 export type SearchPerform<
   $InputData extends DefaultInputData = DefaultInputData,
   $Return extends {} = {},
-> = (z: ZObject, bundle: Bundle<$InputData>) => $Return[] | Promise<$Return[]>;
+> = (z: ZObject, bundle: Bundle<$InputData>) => $Return[] | $Return | Promise<$Return[] | $Return>;
```

Comment:
Shouldn't this be:

```suggestion
> = (z: ZObject, bundle: Bundle<$InputData>) => $Return[] | { results: $Return[], paging_token: string } | Promise<$Return[] | { results: $Return[], paging_token: string }>;
```

Likely would benefit from a helper type to be less verbose.